lib/context-base.js

```diff
@@ -29,3 +39,7 @@ ContextBase.prototype.getScope = function() {
     method.ctor ||
     method.sharedMethod && method.sharedMethod.ctor;
 };
+
+ContextBase.prototype.setReturnArgByName = function(name, value) {
```

Comment:
>Well, I want to make it clear in this file what are the method arguments (name, value). In that sense, each empty method in ContextBase class is different, none of them is a generic NOOP.

That makes sense; you may also consider a named function:
```
ContextBase.prototype.setReturnArgByName = function NOOP(name, value) {}
```
Please feel free to ignore it.